_health-care/_js/_components/error-message.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+/**
+ * An error message to be included conditionally by question components in an error state.
+ *
+ * @constructor
+ */
+class ErrorMessage extends React.Component {
+  render() {
+    return (
+      <span className="usa-input-error-message" id="input-error-message" role="alert">{this.props.message}</span>
```

Comment:
Why can't we just do in question:
```
<div>
   <ErrorMessage idPrefix={_.uniqueId('something')} />
   <ErrorMessage idPrefix={_.uniqueId('something')} />
</div>
```
?
The Question component would have to ensure no colliding IDs, but it'd have to do that anyways if it was directly creating `<input>` components.